Header header 

uint32[] id                  # 此特征线ID

float32[] startPointX        # 此特征线起点畸变补偿后的像素坐标u
float32[] startPointY        # 此特征线起点畸变补偿后的像素坐标v

float32[] endPointX          # 此特征线终点畸变补偿后的像素坐标u
float32[] endPointY          # 此特征线终点畸变补偿后的像素坐标v
